vue.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
-  chainWebpack: config => {
-    config.externals(["vue", "vue-router", /^@gz\/.+/]);
+  chainWebpack: (config) => {
+    config.externals([/^@gz\/.+/]);
```

Comment:
Vue 3 currently doesn't have an external build that is compatible with SystemJS (see https://github.com/vuejs/vue-next/issues/1136), whereas v2 has a UMD build. This is an ongoing issue that you can follow here: https://github.com/systemjs/systemjs/issues/2272
The options for this are:

1. bundle vue/vue-router into the application (not ideal in the long run since every vue application would bundle vue/vue-router again)
2. downgrade to vue 2
3. create an @esm-bundle version (which Joel mentions some interest in doing here https://github.com/systemjs/systemjs/issues/2272#issuecomment-723338042) but there isn't anything preventing someone else from doing this